tests/saitests/py3/sai_qos_tests.py

```diff
@@ -5655,48 +5670,51 @@ def runTest(self):
         margin = int(self.test_params['pkts_num_margin']) if self.test_params.get(
             'pkts_num_margin') else 8
 
-        recv_counters_base, _ = sai_thrift_read_port_counters(
-            self.src_client, asic_type, port_list['src'][src_port_id])
-        dst_q_wm_res_bases = [sai_thrift_read_port_watermarks(
-            self.dst_client, port_list['dst'][sid])[0] for sid in dst_port_ids]
-        print("queue watermark base for all port is {}".format(dst_q_wm_res_bases),
-              file=sys.stderr)
-
         try:
             for i in range(len(prio_list)):
+                eprint("DSCP index {}/{}".format(i + 1, len(prio_list)))
```

Comment:
better to only add necessary log to stderr, especially in looper.
that is also reason that we set log_message's argument "to_stderr" as False by default.
```
def log_message(message, level='info', to_stderr=False):
```

In before, there was a bug which output stderr message in infinite loop , cause huge log file ,and pipeline stuck.

log_message() will output all message to ptf log by default, only necessary info was allowed to output to std for triage purpose.
and ptf log is better for debug purpose, because it can be observed in runtime.   stderr need to additionally handle by infra code after testcase complete, then is able to observed.

Reply:
Went ahead and updated all error printing to log_message, as it looks like that is the new planned logging technique rather than the current error printing.